packages/auth/lib/index.d.ts

```diff
@@ -1201,6 +1201,16 @@ export namespace FirebaseAuthTypes {
    * TODO @salakar missing updateCurrentUser
    */
   export class Module extends FirebaseModule {
+    /**
+     * Returns the current tenant Id.
+     *
+     * #### Example
+     *
+     * ```js
+     * const tenantId = firebase.auth().tenantId;
+     * ```
+     */
+    tenantId: string;
```

Comment:
Can this be null if it has never been set? if so:
```suggestion
    tenantId: string | null;
```